model/gen-ai/registry.yaml

```diff
@@ -192,6 +192,37 @@ groups:
         type: string
         brief: Human-readable name of the GenAI agent provided by the application.
         examples: ["Math Tutor", "Fiction Writer"]
+      - id: gen_ai.agent.version
```

Comment:
are there some existing frameworks out there that have a concept of agent version?

Reply:
let's not add version now  then - we can always add attributes when we need them. Also, it's a good question whether they should be supplied in instrumentation scope (when creating tracer/meter) vs in span attributes.